### PR TITLE
fix(theme): fix some styling for the editor

### PR DESCRIPTION
**Pull request by SamVerschueren**
While documenting the editor, I found some things that couldn't be styled. And I also renamed a couple. I'll comment inline.

### CODE REVIEW

File: packages/components/react/src/CodeMirrorEditor/cm-theme.ts [hunk 1]
```diff
@@ -128,9 +128,11 @@ export const editorTheme = EditorView.theme({
   '.cm-tooltip': {
     background: 'var(--cm-tooltip-backgroundColor)',
     borderColor: 'var(--cm-tooltip-borderColor)',
+    color: 'var(--cm-tooltip-textColor)',
   },
   '.cm-tooltip.cm-tooltip-autocomplete ul li[aria-selected]': {
     background: 'var(--cm-tooltip-backgroundColorSelected)',
+    color: 'var(--cm-tooltip-textColorSelected)',
```
Comment (SamVerschueren) [inline packages/components/react/src/CodeMirrorEditor/cm-theme.ts:135]: It was not possible to set the text color for the text in the tooltip.

File: packages/components/react/src/styles/cm.css [hunk 2]
```diff
@@ -16,16 +16,16 @@
 
   /* Cursor */
   --cm-cursor-width: 2px;
-  --cm-cursor-backgroundColor: var(--tk-elements-editor-cursor-textColor, var(--tk-text-primary));
+  --cm-cursor-backgroundColor: var(--tk-elements-editor-cursorColor, var(--tk-text-primary));
```
Comment (SamVerschueren) [inline packages/components/react/src/styles/cm.css:19]: As discussed, `cursorColor` makes more sense.

File: packages/components/react/src/styles/cm.css [hunk 3]
```diff
@@ -16,16 +16,16 @@
...
 
   /* Selection */
-  --cm-selection-backgroundColorFocused: var(--tk-elements-editor-selection-backgroundColorFocused, #42b4ff);
-  --cm-selection-backgroundOpacityFocused: var(--tk-elements-editor-selection-backgroundOpacityFocused, 0.3);
-  --cm-selection-backgroundColorBlured: var(--tk-elements-editor-selection-backgroundColorBlured, #c9e9ff);
-  --cm-selection-backgroundOpacityBlured: var(--tk-elements-editor-selection-backgroundOpacityBlured, 0.3);
+  --cm-selection-backgroundColorFocused: var(--tk-elements-editor-selection-backgroundColor, #42b4ff);
+  --cm-selection-backgroundOpacityFocused: var(--tk-elements-editor-selection-backgroundOpacity, 0.3);
+  --cm-selection-backgroundColorBlured: var(--tk-elements-editor-selection-inactiveBackgroundColor, #c9e9ff);
+  --cm-selection-backgroundOpacityBlured: var(--tk-elements-editor-selection-inactiveBackgroundOpacity, 0.3);
```
Comment (SamVerschueren) [inline packages/components/react/src/styles/cm.css:28]: I renamed these to match a bit closer to how the terminal does the styling for the selection (`--tk-elements-terminal-selection-backgroundColor`). And renamed `blured` to `inactive` which matches more closely with others which we named `activeBackgroundColor` for instance.

Feel free to make suggestions, or we can also revert them.

**Review by Nemikolh — APPROVED** [review]
Looks really good! :tada:

**Pull request merged**